rts/motoko-rts/src/bigint.rs

```diff
@@ -85,10 +89,14 @@ pub unsafe fn mp_calloc<M: Memory>(
 #[ic_mem_fn]
 pub unsafe fn mp_realloc<M: Memory>(
     mem: &mut M,
-    ptr: *mut libc::c_void,
-    old_size: Bytes<u32>,
-    new_size: Bytes<u32>,
-) -> *mut libc::c_void {
+    ptr: *mut c_void,
+    old_size: Bytes<usize>,
+    new_size: Bytes<usize>,
+) -> *mut c_void {
+    // TODO: Remove these temporary downcasts during 64-bit port
+    let old_size = Bytes(old_size.as_usize());
+    let new_size = Bytes(new_size.as_usize());
```

Comment:
Are these just nops now?

Reply:
Thanks for having spotted this redundant code (I missed the `TODO` above saying that this can be removed).
I deleted these two lines of code.